kdl_parser_py/package.xml

```diff
@@ -28,6 +29,12 @@
   <build_export_depend>urdfdom_py</build_export_depend>
   <build_export_depend>python_orocos_kdl</build_export_depend>
 
+  <!-- The build_depend on urdfdom_py is needed for the doc job; we
+       don't have one for python_orocos_kdl because it is not a
+       catkin package.
+  -->
+  <build_depend>urdfdom_py</build_depend>
```

Comment:
What is `urdfdom_py` needed for at build time of this package?

Reply:
Only because we `find_package` it in `CMakeLists.txt` (and catkin complains if you `find_package` without it being a build_depend).  Should I just remove the `find_package` for it in `CMakeLists.txt`?

Reply:
I don't know why it would need to be `find_package`-ed.